people = {'john': 32, 'rob': 45, 'tim': 20}

print(people.get("rob"))
print(people['rob'])

print(people.get("jim"))#this show there is none with this key
print(people['jim'])#this shows error